tests/workspace_test.py

```diff
@@ -136,3 +134,31 @@ def test_reopening_mode(tmp_path):
         with pytest.warns(UserWarning, match="Closing the workspace in mode 'r'"):
             with fetch_active_workspace(workspace, mode="r+"):
                 assert workspace.geoh5.mode == "r+"
+
+
+def test_in_memory_to_disk():
+    workspace = Workspace()
+    points = Points.create(workspace, vertices=np.random.randn(12, 3), name="Points_A")
+
+    with Workspace() as out_workspace:
+        new_points = points.copy(parent=out_workspace)
+
+    with pytest.raises(Geoh5FileClosedError):
+        compare_entities(points, new_points, ignore=["_parent"])
+
+    with out_workspace.open():
+        compare_entities(points, new_points, ignore=["_parent"])
+
+    workspace.close()
+
+    # with pytest.raises(
```

Comment:
**suggestion (maintainability):** remove code commented out (or comment why it was commented out)

Reply:
Done